torch/csrc/jit/codegen/fuser/cuda/resource_strings.h

```diff
@@ -13,6 +13,8 @@ tensor as input. Correct code for this case is generated, however, nvrtc does
 not know how to handle int*_t integer types, so typedefs help it handle those
 cases*/
 
+static constexpr auto bfloat16_type_string = "__nv_bfloat16";
```

Comment:
Can you please add short explanation to PR description, why this is needed?

Reply:
@malfet Description updated